### DISCUSSION TITLE
\Http\Browser: Is there a way to get the content on request errors?

### DISCUSSION THREAD

**Discussion by Donatello-za** [reactphp/reactphp#526] [Q&A]
When a request is made to the server such as using the `post()` method on a `\Http\Browser` instance and the server returns with an error 400 response, is there a way to still get the body of the response? Typically the Browser instance will throw an exception with a generic error such as "HTTP status code 400 (Bad Request)" but I can't figure out how to still get the body of the response in order to log or debug it.

**Comment by WyriHaximus** [comment]
Have a look at https://reactphp.org/http/#withrejecterrorresponse it describes how not to throw on non-successful codes. As well as how to get te response from the exception.
